go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/aws/karpenter
 
-go 1.19
```

Comment:
This doesn't make sense in a `go.mod` file format. This should stay as-is.

Reply:
The go vulnerability that was causing checks to fail was failing because this PR was delayed after the new go version drop: https://github.com/actions/go-versions/pull/60

Reply:
Got it, version-manifest.json was missing the new go version in GitHub Actions/go-versions repo,  reverted the change.